packages/extension-chakra-storefront/src/theme/components/base/button.js

```diff
@@ -21,16 +21,17 @@ const baseLinkStyles = {
 
 const button = defineRecipe({
     base: {
-        borderRadius: 'sm'
+        borderRadius: 'sm',
+        colorPalette: 'blue'
```

Comment:
For the Button, setting the default color palette to blue. This behaviour was also in the original code.